db/migrate/20201117125417_add_approved_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddApprovedToUsers < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :approved, :boolean
+    change_column_default :users, :approved, false
```

Comment:
`add_column :users, : approved, :boolean, :default => true` would not work? if yes, #change method should be enough

Reply:
strong_migration gem does not miss such migration